src/common/tusb_compiler.h

```diff
@@ -56,7 +56,7 @@
 #elif defined (__STDC_VERSION__) && __STDC_VERSION__ >= 201112L
   #define TU_VERIFY_STATIC   _Static_assert
 #elif defined(__CCRX__)
-  #define TU_VERIFY_STATIC(const_expr, _mess) typedef char TU_XSTRCAT(Line, __LINE__)[(const_expr) ? 1 : 0];
+  #define TU_VERIFY_STATIC(const_expr, _mess) typedef char TU_XSTRCAT3(Line, __LINE__, _TU_COUNTER_)[(const_expr) ? 1 : 0];
```

Comment:
This is the macro that expands to something like:

`typedef char Line<XY>[(const_expr) ? 1 : 0];`

This commit changes the macro to:

`typedef char Line<XY><Z>[(const_expr) ? 1 : 0];`
With Z being the value of `_TU_COUNTER_`.

Yes, the number does not contain the exact line number (because `_TU_COUNTER_` was appended). But this is not an issue since the compiler error generated in case of a failed verification is the following:
`E0520094: The size of an array must be greater than zero`
 
Meaning the line index was unused anyways.

Reply:
thanks, actually we don't need the line for indicator, gcc would be able to emit the exact location. We can just replace __LINE__ with __TU_COUNTER__ to make it consistent  with the followed ``#else`.